http4s-client-blaze-pureconfig/src/main/scala/com/avast/sst/http4s/client/pureconfig/package.scala

```diff
@@ -0,0 +1,3 @@
+package com.avast.sst.http4s.client
+
+package object pureconfig extends ConfigReaders
```

Comment:
shouldn't the name of the object which contains implicits be `implicits`, not `pureconfig` (as is the custom in the TypeLevel world)?

Reply:
You mean `pureconfig.implicits._` or just `implicits._`. If it was just `implicits._` I wonder what happens if I have two modules with the same `package object`? Does that even work?

Reply:
in this case, I think it should be `com.avast.sst.http4s.client.pureconfig.implicits`. Then the use would do
```scala
import com.avast.sst.http4s.client.pureconfig.implicits._
```
does that make sense?